go.mod

```diff
@@ -5,7 +5,7 @@ go 1.13
 require (
 	github.com/cloudevents/sdk-go v1.1.2
 	github.com/dghubble/sling v1.3.0
-	github.com/edgexfoundry/app-functions-sdk-go v1.0.1-dev.13
-	github.com/edgexfoundry/go-mod-core-contracts v0.1.52
-	github.com/stretchr/testify v1.4.0
+	github.com/edgexfoundry/app-functions-sdk-go v1.0.1-dev.35
```

Comment:
Change to `v1.0.1-dev.40`, but soon to be `v1.0.1-dev.41` with fix Brian is working on.